poh-bundle-linux-x64/oh.sh

```diff
@@ -1,68 +1,259 @@
 #!/bin/bash
+#
+# Open Hospital (www.open-hospital.org)
+# Copyright © 2006-2020 Informatici Senza Frontiere (info@informaticisenzafrontiere.org)
+#
+# Open Hospital is a free and open source software for healthcare data management.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# https://www.gnu.org/licenses/gpl-3.0-standalone.html
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+#
+
+######## Environment check:
+
+######## OPENHOSPITAL Configuration
+# POH_PATH is the directory where Portable OpenHospital files are located
+# POH_PATH=/usr/local/PortableOpenHospital
+
+# name of this shell script
+SCRIPT_NAME="oh.sh"
```

Comment:
```suggestion
SCRIPT_NAME=$(basename "$0")
```

as you did below for the usage message